schema/operating_rules.json

```diff
@@ -0,0 +1,113 @@
+      ]
+    },
+    "data": {
+      "type": "object",
+      "required": [
+        "operating_rules"
+      ],
+      "properties": {
+        "operating_rules": {
+          "type": "array",
+          "description": "Array that contains one object per operating rule as defined below.",
+          "items": {
+            "type": "object",
+            "title": "A Schema",
+            "required": [
+              "from_zone_id",
+              "to_zone_id",
+              "calendars",
+              "vehicle_type_id"
+            ],
+            "properties": {
+              "from_zone_id": {
+                "type": "string",
+                "title": "ID from a zone defined in zones.json representing the boarding zone for the current rule."
+              },
+              "to_zone_id": {
+                "type": "string",
+                "title": "ID from a zone defined in zones.json representing the alighting zone for the current rule. from_zone_id and to_zone_id MAY reference the same zone."
+              },
+              "start_pickup_window": {
+                "type": "string",
+                "title": "Time at which the pickup starts being available in from_zone_id defined in this array. If start_pickup_window is provided, either end_pickup_window or end_dropoff_window MUST also be provided."
+              },
+              "end_pickup_window": {
+                "type": "string",
+                "title": "Time at which the pickup stops being available in from_zone_id defined in this array. If end_pickup_window is provided, start_pickup_window MUST be provided."
+              },
+              "calendars": {
+                "type": "array",
+                "title": "Array of calendar IDs from calendars.json defining the dates and days when the pickup and drop off occur.",
+                "items": {
+                  "type": "string",
+                  "title": "A Schema",
+                  "examples": [
+                    "all-days"
+                  ]
+                }
+              },
+              "end_dropoff_window": {
+                "type": "string"
+              },
+              "brand_id": {
+                "type": "string"
+              },
+              "vehicle_type_id": {
+                "description": "Array of vehicle types used for delivering the on-demand service.",
+                "type": "array"
+              },
+              "fare_id": {
+                "type": "string"
+              }
+            },
+            "examples": [
+              {
+                "from_zone_id": "zoneA",
+                "to_zone_id": "zoneA",
+                "start_pickup_window": "06:00:00",
+                "end_pickup_window": "09:00:00",
+                "end_dropoff_window": "09:30:00",
+                "calendars": [
+                  "weekend",
+                  "labor_day"
+                ],
+                "brand_id": "large_ride",
+                "vehicle_type_id": "large_van",
```

Comment:
In the spec, this field is listed as an Array, but our own example shows it as a string. Can we update the example so that it's up to spec?

Reply:
Good catch, resolved in 1aa6415b07655d75f5fe092a401f3a316ece34b2

Reply:
Just updated in `reference.md` as well